### PR TITLE
[FIX] connector_acp_ftp: Change to confirmed on successful connection test

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: connector_acp_ftp/models/backend_acp.py [hunk 1]
```diff
@@ -61,8 +61,7 @@ def action_confirm(self):
         if self.connection_type == "ftp":
             self._ftp_upload(None)
             return True
```
Comment (max3903) [inline connector_acp_ftp/models/backend_acp.py]:
```suggestion
```

**Review by max3903 — APPROVED** [review]

File: connector_acp_ftp/models/backend_acp.py [hunk 2]
```diff
@@ -61,8 +61,7 @@ def action_confirm(self):
...
-        else:
-            return super().action_confirm()
+        return super().action_confirm()
```
Comment (max3903) [inline connector_acp_ftp/models/backend_acp.py:61]: @dreispt If the FTP connection works, True will be returned and the state will not be updated.

```
        if self.connection_type == "ftp":
￼            self._ftp_upload(None)
￼            return super().action_confirm()
```

Comment (dreispt) [reply connector_acp_ftp/models/backend_acp.py:61]: Good point. But I think the solution is to remove the return statement inside the if block.

**Review by max3903 — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Pull request merged**